frontend/src/tests/test_data.js

```diff
@@ -0,0 +1,258 @@
+import instructorsJSON from "./samples/instructors.json";
+import applicantsJSON from "./samples/applicants.json";
+import positionJSON from "./samples/positions.json";
+import wrong1positionsJSON from "./samples/wrong1positions.json";
+import wrong3positionsJSON from "./samples/wrong3positions.json";
+import assignmentsJSON from "./samples/assignments.json";
+import ddahsJSON from "./samples/ddahs.json";
```

Comment:
`samples` is not such a good name. `test-data` or `testing-data` or something would be better. Also did you look into Jest `snapshots`?

Reply:
I was looking at [https://jestjs.io/docs/en/snapshot-testing](url) about snapshots. I am kind of confused about how this should work in the case of our import from file test. I think the manual comparison of contents is enough maybe? Can you give me a hint on how I may utilize snapshots?